tests/test_dummy.py

```diff
@@ -625,7 +625,7 @@ def test_parameterize_and_draw_ixns():
     """
 
     ff_handlers = deserialize_handlers(open("timemachine/ff/params/smirnoff_1_1_0_sc.py").read())
-    ff = Forcefield(ff_handlers)
+    ff = Forcefield.from_handlers(ff_handlers)
```

Comment:
This is a breaking API change; I didn't immediately see any usage of the `Forcefield.__init__` outside of the main repo, but might be good to double-check